test/receiveAndDeleteMode.spec.ts

```diff
@@ -171,6 +175,9 @@ describe("Streaming Receiver from Queue/Subscription", function(): void {
     should.equal(receivedMsgs[0].messageId, testMessages[0].messageId);
 
     await receiveListener.stop();
+    if (errorFromErrorHandler) {
+      should.fail(!!errorFromErrorHandler, false, errorFromErrorHandler.message);
```

Comment:
The intent seems to be to simply fail the test if there's an error.

We can use chai.assert.fail(uncaughtErrorFromHandlers.message) and avoid the true, false comparison (and potential bugs from interpreting what the fail is supposed to do).

should.fail() seems generally confusing, I was expecting it to have just one input param - the failing message (like how assert.fail does) but *shrugs*

Reply:
Good point! Updated to use assert.fail()